sympy/stats/joint_rv.py

```diff
@@ -101,7 +101,6 @@ def marginal_distribution(self, *indices):
         if self.distribution.is_Continuous:
             f = Lambda(sym, integrate(self.distribution(*all_syms), *limits))
         elif self.distribution.is_Discrete:
-            limits = [(limit[0], limit[1].inf, limit[1].sup) for limit in limits]
```

Comment:
What result did this change? If none, why?

Reply:
Actually I did this when `Sum` didn't accept `Range` as limits in an earlier PR but now it isn't needed so removing this line.